x-pack/legacy/plugins/uptime/public/lib/helper/observability_integration/get_metrics_href.ts

```diff
@@ -46,7 +46,7 @@ export const getInfraIpHref = (summary: MonitorSummary, basePath: string) => {
       const expression = encodeURIComponent(`host.ip : ${value}`);
       return addBasePath(
         basePath,
-        `/app/infra#/infrastructure/inventory?waffleFilter=(expression:'${expression}',kind:kuery)`
+        `/app/infra#/metrics/inventory?waffleFilter=(expression:'${expression}',kind:kuery)`
```

Comment:
Do these links work for you? In my local instance they get automatically changed to something like `/app/infra#/infrastructure/inventory...`